rocev2/test/src/scala/rdma/RecvQTest.scala

```diff
@@ -0,0 +1,193 @@
+package rdma
+
+import org.scalatest.funsuite.AnyFunSuite
+import rdma.RdmaConstants.PSN_WIDTH
+import spinal.core._
+import spinal.core.sim._
+import spinal.lib._
+
+import scala.collection.mutable
+import scala.util.Random
+
+class SeqOutTest extends AnyFunSuite {
+
+  class RxDriver(
+      rx: Stream[Fragment[RdmaDataBus]],
+      clockDomain: ClockDomain,
+      opCode: OpCode.OpCode,
+      fragments: Int
+  ) {
+    val psnQueue = new mutable.Queue[Int]
+    rx.valid #= false
+    rx.last #= false
+    rx.bth.opcodeFull #= opCode.id
+
+    def push(psnList: List[Int]): Unit = {
+      psnQueue ++= psnList
+    }
+
+    def clear(): Unit = {
+      psnQueue.clear()
+    }
+
+    val driver = fork {
```

Comment:
use def?

Reply:
this fork block auto start when the class be created, I don't mean to call it manually. so let me delete the assign from fork to a val.